### PR TITLE
Get throwable instead of get exception

### PR DESCRIPTION
**Pull request by Bl00D4NGEL**
Refs: https://github.com/rectorphp/rector-symfony/issues/29
Fixes: https://github.com/rectorphp/rector-symfony/issues/29

### CODE REVIEW

**Comment by Bl00D4NGEL** [comment]
@TomasVotruba please review (the PHPStan problems don't seem related to my fix)

File: config/sets/symfony/symfony43.php [hunk 1]
```diff
@@ -27,6 +27,11 @@
             RenameMethodRector::METHOD_CALL_RENAMES => ValueObjectInliner::inline([
                 new MethodCallRename('Symfony\Component\BrowserKit\Response', 'getStatus', 'getStatusCode'),
                 new MethodCallRename('Symfony\Component\Security\Http\Firewall', 'handleRequest', 'callListeners'),
+                new MethodCallRename(
```
Comment (TomasVotruba) [inline config/sets/symfony/symfony43.php:30]: Could you link the rename in Symfony with a comment here? Just so we know the origin of this change in Symfony

```php
// @see https://...
```

Comment (Bl00D4NGEL) [reply config/sets/symfony/symfony43.php:31]: Done that :)

Comment (TomasVotruba) [reply config/sets/symfony/symfony43.php:31]: Thanks :+1: 

What about its sister?

https://github.com/symfony/http-kernel/blob/801b925e308518ddf821ba91952c41ae77c77507/Event/GetResponseForExceptionEvent.php#L67

**Comment by TomasVotruba** [comment]
I'll fix the php-parser changes, it's caused by php-parser 4.13.0 release. Don't worry about it :slightly_smiling_face:

**Comment by TomasVotruba** [comment]
I've added PHPStan fixes. Could you rebase on `main`?

File: config/sets/symfony/symfony43.php (hunk 1 above)
Comment (Bl00D4NGEL) [reply config/sets/symfony/symfony43.php:31]: Good call! Fixed that, too :)

**Comment by TomasVotruba** [comment]
Thanks :+1: 

PHPStan errors are unrelated to your changes. Let's ship it

**Comment by mglaman** [comment]
Woo! Thanks, ya'll!

**Pull request merged**